src/main/java/calculator/Calcurator.java

```diff
@@ -0,0 +1,63 @@
+package calculator;
+
+import java.util.Arrays;
+
+public class Calcurator {
+
+    public int calcurateString(String expression) {
+        String[] splitStr = expression.split(" ");
+        int result=0;
+        result = toInt(splitStr[0]);
+        for( int i=0; i<splitStr.length -2; i+=2){
+            char operator = operatorStringToChar( splitStr[i+1]);
+            int secondValue = toInt(splitStr[i+2]);
+            result = calcurate( result, operator, secondValue);
```

Comment:
단항 연산자(++ 혹은 --)의 경우와 .을 제외한 모든 이항(binary) 연산자는 연산수들과는 공백으로 분리되어져야 합니다 :) 또한 for문 사용시 다음과 같이 괄호 사이에 공백을 넣어주시는게 컨벤션이오니 참고 부탁드려요!

```suggestion
        int result = 0;
        result = toInt(splitStr[0]);
        for (int i = 0; i < splitStr.length - 2; i += 2) {
            char operator = operatorStringToChar(splitStr[i + 1]);
            int secondValue = toInt(splitStr[i + 2]);
            result = calcurate(result, operator, secondValue);
```